src/features/todos.ts

```diff
@@ -1,8 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set: (_, action) => {
+      return action.payload;
```

Comment:
The reducer function 'set' is using an underscore '_' for the first parameter, which is typically used to indicate that the parameter is unused. However, in Redux reducers, the first parameter is usually the current state, and it should be used to return the new state. Consider renaming '_' to 'state' and using it appropriately.